lib/PoolsClosed/machines.rb

```diff
@@ -10,6 +10,8 @@ class Machines
     RLSD     = 'poolsclosed_releasedmachines'.freeze
     ERR      = 'poolsclosed_lasterror'.freeze
 
+    attr_reader :jobs
+
     def initialize(cnf)
```

Comment:
attr_reader makes the jobs var accessible to rspec (or whatever external class). Is that what you're asking?